wallet/tutorials/react-dapp-local-state.md

```diff
@@ -166,7 +166,7 @@ However, MetaMask provides the
 [@metamask/detect-provider](https://github.com/MetaMask/detect-provider) module to detect the
 MetaMask Ethereum provider or any provider injected at `window.ethereum` on any platform or browser.
 
-We recommend [using `@metamask/detect-provider`](../how-to/connect/detect-metamask.md)
+We recommend [using `@metamask/detect-provider`](javascript-dapp-simple.md)
```

Comment:
```suggestion
You can use [`@metamask/detect-provider`](https://github.com/MetaMask/detect-provider)
```

It's in the backlog to update this tutorial with EIP-6963 instructions, but we should avoid recommending detect-provider currently.